tests/Dockerfile.tests

```diff
@@ -1,4 +1,4 @@
-FROM registry.access.redhat.com/ubi8/go-toolset:1.17 as builder
+FROM registry.access.redhat.com/ubi8/go-toolset:1.18 as builder
```

Comment:
Same as above. If we aren't constrained by the container image here, 1.19 could be worth considering.